RELEASE.md

```diff
@@ -9,33 +9,22 @@ necessary.
 
 Make sure there aren't any pending issues before tagging a new release.
 
-## Release commit
+## Release script
 
-### Change version in `Dockerfile`
+Run `make-release.sh vX.Y.Z`. The script will change the image version
+specified in [`Dockerfile`](./Dockerfile), change the version in
+[CHANGES](./CHANGES.md) and add a release description, create a release commit
+in a new release branch and update the version used in
+[Dockerfile](./Dockerfile) to `vX.Y.Z-dev` to avoid usage of the `main` branch
+in actual deployments.
```

Comment:
The release script is used to make the necessary changes to enable tagging version `vX.Y.Z`:

```
$ ./make-release.sh vX.Y.Z
```

The script will create a separate branch, where it will make the necessary changes and prompt you for any further required information.